.gitignore

```diff
@@ -42,11 +43,8 @@ inventory_url.yaml
 analytics/config/testFiles/
 analytics/filesystem/testFiles/
 
-# autogenerated version file
-# static/version.txt
-
-.idea/
-.vscode/
+# autogenerated files
+prebid-server.iml
```

Comment:
What's an`iml` file?

Reply:
IDEA module file

https://www.jetbrains.com/help/idea/creating-and-managing-modules.html